card.go

```diff
@@ -24,8 +24,8 @@ type CardFunding string
 // For more details see https://stripe.com/docs/api#create_card and https://stripe.com/docs/api#update_card.
 type CardParams struct {
 	Params
-	Token                                         string
-	Customer, Recipient                           string
+	External_Account, Token                       string
```

Comment:
When I remove the External_Account Param I get the follow error response:

{"type":"invalid_request_error","message":"Missing required param: external_account.","param":"external_account","request_id":"req_foo_mustache","status":400}

When I use that param, I get a successful request.